parts/k8s/cloud-init/artifacts/cse_install.sh

```diff
@@ -298,8 +304,15 @@ pullContainerImage() {
 
 cleanUpContainerImages() {
     # TODO remove all unused container images at runtime
+    # CUSTOM_CLOUD_SUFFIX only set when extracting custom Hyperkube builds (e.x.: Azure Stack)
+    CUSTOM_CLOUD_SUFFIX=$1
     docker rmi $(docker images --format '{{.Repository}}:{{.Tag}}' | grep -v ${KUBERNETES_VERSION} | grep 'hyperkube') &
     docker rmi $(docker images --format '{{.Repository}}:{{.Tag}}' | grep -v ${KUBERNETES_VERSION} | grep 'cloud-controller-manager') &
+    if [[ "${CUSTOM_CLOUD_SUFFIX}" == "${AZURE_STACK_SUFFIX}" ]]; then
```

Comment:
Can we check the value of `TARGET_ENVIRONMENT` here instead of setting a `CUSTOM_CLOUD_SUFFIX` arg?